.azure-pipelines/testscripts_analyse/analyse_testcases.py

```diff
@@ -0,0 +1,102 @@
+"""
+To ensure that a test script is included in the PR checker for the corresponding topology type,
+this script will perform a validation check.
+Additionally, the return value will be enhanced to include more detailed information, such as the category of the test.
+Post-execution, the script will also append the scan time and track ID to the results.
+
+The return value is formatted as below:
+[
+    {
+        'testscript': 'acl/custom_acl_table/test_custom_acl_table.py',
+        'topology': 't0',
+        'trackid': '3aa57f0f-8f18-4cf7-ae1e-0a18973a0b86',
+        'scantime': '2024-05-3106: 53: 40.826349',
```

Comment:
Why the value of `scantime` contains some space? Is this correct?

Reply:
Fixed